analysis_options.yaml

```diff
@@ -7,8 +7,6 @@
 
 # The following line activates a set of recommended lints for Flutter apps,
 # packages, and plugins designed to encourage good coding practices.
-include: package:flutter_lints/flutter.yaml
```

Comment:
why you remove that?

Reply:
It's not necessary here

Reply:
We need to have lint check, IMO we need it :)

Reply:
The problem here is that we have not added library lint so it is always not found when doing `flutter analyze`

Reply:
okie, move forward, we will back on that problem later : #367